src/log.js

```diff
@@ -64,6 +64,27 @@ export function setReportError(fn) {
 }
 
 /**
+ * An extensible cross-browser error class that correctly supports messages,
+ * stack traces, and instanceof for subclasses.
+ * @abstract
+ */
+export class AbstractError extends Error {
+  /**
+   * @param {string} message
+   */
+  constructor(message) {
+    super(message);
```

Comment:
Danger! This may or may not behave as you expect based on how compliant the browser is.